gitogether/1st-meeting/isfha.html

```diff
@@ -6,6 +6,9 @@
 </head>
 <body>
 	<main>
+		<figure>
+			<img src="https://avatars2.githubusercontent.com/u/6775159?s=460&v=4" alt="avatar">
```

Comment:
you have a lot of free time, so you have to make things that are not useful like this 🤣